codegen/template.go

```diff
@@ -93,6 +94,10 @@ func unref(t string) string {
 	return t
 }
 
+func args(v ...interface{}) []interface{} {
```

Comment:
mm ... is this used in templates? Why array instead of kv pair, is order matter?

Reply:
Yes, it's used. This function combines variadic params into one param because the `template` function for nested templates only takes one param.